packages/flutter_tools/lib/src/cache.dart

```diff
@@ -62,10 +62,11 @@ class Cache {
         locked = true;
       } on FileSystemException {
         if (!printed) {
-          printStatus('Waiting to be able to obtain lock of Flutter binary artifacts directory...');
+          printTrace('Waiting to be able to obtain lock of Flutter binary artifacts directory: ${_lock.path}');
+          printStatus('Waiting for another flutter command to finish...');
```

Comment:
My goal was to have the message suggest a solution. "lock of Flutter binary artifacts directory" talks about an implementation detail. A user won't know what to do with it. "another flutter command" OTOH means that perhaps you could Ctrl + C in another terminal window. "finish" does not necessarily mean that it has to exit, but only to finish doing what it's doing that's blocking the current command. While it's not precise, it is not wrong.